### DISCUSSION TITLE
Google Webview (Sign Out exploit)

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1276] [Webview JOIN TITANIUM NETWORK TO SHARE PLEASE]
Instructions:
1. Sign out of your (Chromebook)device from another device logged in to the same account. from Google manage account security
2. Go to tinyurl.com/addsession , to sign back in to make sure you still access website to do with Google.
3. You will get a notifacation like this 
image link = https://github.com/user-attachments/assets/4985081c-3963-497f-8451-3bc9fb46037b
4. Tap on it and it should open this
image link = https://github.com/user-attachments/assets/fa4a3c0e-7a84-48a0-b622-4b0ad37fa3aa
5. It should load and then look like this 
https://github.com/user-attachments/assets/1b0c94d3-5e89-4e19-ac46-aa7366d6e70c
6. Turn off your wifi and hit next. It should then go to a page that says something went wrong with a next button.
7. Turn your wifi back on and make sure it is connected then hit next.
8. It should look like this 
image link = https://github.com/user-attachments/assets/f094c932-d48e-44e7-b104-db2feae481ea
9.  Tap on the Google logo and  it should take you to Google search. Now you have an unblocked browser with no history saving. Note: Use the arrow in the top left to go back pages

if somebody make this a actual exploit pls give me credit

**Comment by akabutnicer** [comment]
can we start tending to just say webview instead of,  'UNBLOCKED BROWSER NO HISTORY'? it's so misleading 💀

**Reply by ghost** [reply]
sorry

**Comment by akabutnicer** [comment]
also, it's too many steps for such a small outcome

**Reply by ghost** [reply]
sorry

**Comment by ghost** [comment]
sorry for the name , I was confused and Im dumb

**Reply by awsome-webdev** [reply]
its better than no exploit

**Reply by akabutnicer** [reply]
fair enough
